XmlSchemaClassGenerator/TypeModel.cs

```diff
@@ -739,16 +742,9 @@ public void AddMembersTo(CodeTypeDeclaration typeDeclaration, bool withDataBindi
                     member.Type = typeReference;
                 }
 
-                if (backingField != null)
-                {
-                    var propertyValueTypeCode = IsCollection || isArray ? PropertyValueTypeCode.Array : propertyType.GetPropertyValueTypeCode();
-                    member.Name += GetAccessors(member.Name, backingField.Name, propertyValueTypeCode, isPrivateSetter, withDataBinding);
-                }
-                else
-                {
-                    var privateSetter = isPrivateSetter ? "private " : string.Empty;
-                    member.Name += $" {{ get; {privateSetter}set; }}"; // hack to generate automatic property
-                }
+                member.Name += backingField != null

```

Comment:
Can you revert this? Sonarcloud complains about the nested ternary operator and I think they have a point.